### PR TITLE
Add support for python3 installation of 'endpoint instance show'.

### PR DESCRIPTION
**Pull request by atoaca**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -12,3 +12,4 @@ oslo.i18n>=2.1.0 # Apache-2.0
 oslo.utils>=3.5.0 # Apache-2.0
 requests!=2.9.0,>=2.8.1 # Apache-2.0
 stevedore>=1.5.0 # Apache-2.0
+future
```
Comment (aznashwan) [inline requirements.txt:15]: Why have this as a requirement if it's not directly imported anywhere?

**Comment by aznashwan** [comment]
Any updates here?

File: requirements.txt (hunk 1 above)
Comment (atoaca) [reply requirements.txt:15]: the `builtins` import originates from the `future` package.

**Comment by atoaca** [comment]
> Any updates here?

Retested, works.

File: requirements.txt (hunk 1 above)
Comment (aznashwan) [reply requirements.txt:15]: Thanks for the clarification, I basically only just used `six` and wasn't aware how `future` is structured.

**Pull request merged**